app/models/location.rb

```diff
@@ -20,6 +20,7 @@ class Location < ApplicationRecord
   NOMIS_TYPES_WITH_DOCUMENTS = %w[STC SCH].freeze
 
   has_and_belongs_to_many :suppliers
+  has_and_belongs_to_many :regions
```

Comment:
does a location actually belong to more than one region? I can't seem to see an example of that in the yaml file?

Reply:
You have to look really closely. Annoyingly, there are only 2 locations that exist in 2 regions so it can't be just mapped as a simple `belongs_to` relationship. They're `FMI` and `PRI`.

Reply:
jolly good 👍